src/Components/DataHistoryExplorer/Internal/TimeSeriesTwinCallout/TimeSeriesTwinCallout.stories.local.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import { getDefaultStoryDecorator } from '../../../../Models/Services/StoryUtilities';
+import TimeSeriesTwinCallout from './TimeSeriesTwinCallout';
+import { ITimeSeriesTwinCalloutProps } from './TimeSeriesTwinCallout.types';
```

Comment:
I noticed that we have stories.local for most of these components. Does every component really make calls to live endpoints?

Reply:
yes?

Reply:
hmm ok. I thought they'd be a bit more encapsulated and only the top parent would be making those fetches. What kind of data are we pushing or pulling from the list of series? I would have expected that to be mostly just rendering a list of items.